3ds Max/Max2Babylon/Exporter/BabylonExporter.IMaterialExporter.cs

```diff
@@ -1,31 +1,73 @@
 using System;
+using System.Collections.Generic;
 using System.Drawing;
 using Autodesk.Max;
+using Babylon2GLTF;
 using BabylonExport.Entities;
-using GLTFExport.Entities;
-
+using GLTFExport.Entities;
+using Utilities;
+
 namespace Max2Babylon
 {
-    
+    internal class MaxGLTFMaterialExporter : IGLTFMaterialExporter

```

Comment:
Should not this be moved in a separate file ? 

Sounds weird in a IMaterialExporter one considering I is for interface ?

Reply:
makes sense, I'll move the implementation to a separate file.